src/microparams-init.c

```diff
@@ -951,7 +951,8 @@ void xnn_init_blockwise_scale_fp32_params(
       const size_t tile_size = channels_tile;
       for (size_t tile_offset = 0; tile_offset < tile_size; tile_offset++) {
         size_t scale_index = (tile_start + tile_offset) * num_blocks + block_start;
-        unaligned_indexed_store_f32(packed_w, tile_offset, scale[scale_index]);
+        // 1/16 because the weight are << 4 in the innermost loop to save a shift
+        unaligned_indexed_store_f32(packed_w, tile_offset, scale[scale_index]/16.0f);
```

Comment:
unaligned_indexed_store_f32(packed_w, tile_offset, scale[scale_index] * (1.0f / 16.0f));
Note I plan to change 4 bit to low bits for qd8

Reply:
Yep, `fdiv` should be avoided here. Compilers aren't smart enough.

Reply:
cc @GregoryComer  - he is taking over this PR.